server/src/main/java/org/apache/druid/server/coordinator/DruidCoordinatorConfig.java

```diff
@@ -92,4 +92,10 @@ public int getHttpLoadQueuePeonBatchSize()
   {
     return 1;
   }
+
+  @Config("druid.coordinator.load.segment.strategy")
+  public String getLoadSegmentStrategy()
+  {
+    return "newFirst";
```

Comment:
shouldn't this be "roundRobin"

Reply:
I named the current strategy of loading newer segments first as `newFirst`. It is also the default strategy.